core/templates/pages/learner-dashboard-page/new-learner-dashboard/community-lessons-tab.component.html

```diff
@@ -0,0 +1,654 @@
+                                          [getLastUpdatedMsec]="tile.lastUpdatedMsec"
+                                          [getObjective]="tile.objective"
+                                          [getCategory]="tile.category"
+                                          [getRatings]="tile.ratings"
+                                          [getNumViews]="tile.numViews"
+                                          [getThumbnailIconUrl]="tile.thumbnailIconUrl"
+                                          [getThumbnailBgColor]="tile.thumbnailBgColor">
+          </oppia-exploration-summary-tile>
+        </div>
+        <div *ngIf="getTileType(tile) === 'collection'">
+          <i class="remove-icon fa fa-times"
+             *ngIf="tile.toggle || checkMobileView()"
+             aria-hidden="true"
+             (click)="openRemoveActivityModal('I18N_LEARNER_DASHBOARD_PLAYLIST_SECTION', 'I18N_DASHBOARD_COLLECTIONS', tile)"
+             ngbTooltip="Remove"
+             container="body"
+             placement="top">
+          </i>
+          <oppia-collection-summary-tile [getCollectionId]="tile.id"
+                                         [getCollectionTitle]="tile.title"
+                                         [getLastUpdatedMsec]="tile.lastUpdatedMsec"
+                                         [getObjective]="tile.objective"
+                                         [getNodeCount]="tile.nodeCount"
+                                         [getCategory]="tile.category"
+                                         [getThumbnailIconUrl]="tile.thumbnailIconUrl"
+                                         [getThumbnailBgColor]="tile.thumbnailBgColor">
+          </oppia-collection-summary-tile>
+        </div>
+      </div>
+      <div *ngIf="checkMobileView() && displayLessonsInPlaylist !== 0">
+        <div class="pagination-bottom">
+          <i class="fa fa-angle-left pagination-arrow" aria-label="Previous Page"
+             (click)="changePageByOne('MOVE_TO_PREV_PAGE', 'playlist')">
+          </i>
+          <span class="dot">
+            <span class="pagination-number">
+              {{pageNumberInPlaylist}}
+            </span>
+          </span>
+          <i class="fa fa-angle-right pagination-arrow" aria-label="Next Page"
+             (click)="changePageByOne('MOVE_TO_NEXT_PAGE', 'playlist')">
+          </i>
+        </div>
+      </div>
+      <div class="oppia-toggle-button-section" *ngIf="totalLessonsInPlaylist.length > 3 && !checkMobileView()">
+        <button class="btn oppia-toggle-btn"
+                (click)="handleShowMore('playlist')"
+                *ngIf="!showMore['playlist']">
+                Show More
+        </button>
+        <button class="btn oppia-toggle-btn"
+                (click)="handleShowMore('playlist')"
+                *ngIf="showMore['playlist']">
+                Show Less
+        </button>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="subscriptions-section">
```

Comment:
Can you add this also to video? You can subscribe to a creator by clicking on 'Author Profiles' at bottom left when playing exp by a creator. (so locally, you'd need 2 accounts)
No need to show the entire steps. Just how it looks in the dashboard.

Reply:
Added screenshots. PTAL!